discovery-provider/alembic/versions/36eac5ed00bf_create_latest_indexing_state.py

```diff
@@ -0,0 +1,28 @@
+"""create latest indexing state
+
+Revision ID: 36eac5ed00bf
+Revises: edccccc274a7
+Create Date: 2021-11-29 22:12:54.657724
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '36eac5ed00bf'
+down_revision = 'edccccc274a7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "latest_indexing_states",
+        sa.Column("tablename", sa.String(), nullable=False),
+        sa.Column("latest_indexing_state", sa.Integer(), nullable=False),
```

Comment:
state here suggests to me this is tracking some state, when it's actually just a checkpoint, maybe just call this `checkpoint`? or table called `indexing_checkpoints` and values `tablename`, `last_index`

wdyt?

Reply:
yup sounds good i'll use that name